vSIM changed to SR-SIM - yml, credentials, configs, gNMIc, etc. Automation panel is not working

tiago-amado opened the pull request:
The setup was changed from vSIM to SR-SIM and its working fine except the automation panel that is returning: "Error 404 - Backend Disconnected". 

I've changed the yml, configs and credentials changed to admin/NokiaSros1!

The rest seems to be working fine: gNMic, Grafana, EdgeShark, ANYsec, ...

hellt commented:
how did you go about sr-sim not capturing packets in both directions?

hellt commented on automation_panel/backend/main.go:
```diff
@@ -326,7 +326,7 @@ func linkSubscribeRequest(group *[]LinkEndpoint) {
 			Port:     "57400",
 			Username: "admin",
-			Password: "admin",
+			Password: "NokiaSros1!",   //# SR-SIM credentials changed to admin/NokiaSros1! (vSIM were admin/admin)
```
regarding backend issues
check the logs of the panel node

```
docker logs panel
```

hellt replied:
ah, you also need to build a panel container

https://github.com/srl-labs/sros-anysec-macsec-lab/tree/main/automation_panel

then for testing you can push to your own account and change the path in the topology to it

tiago-amado commented:
Yes, that's it, you're right. It works now.
I've built the panel and change the yml to use it and its working, except the capture buttons that need to change from PE1 eth1 & eth2 to PE1-iom  e1-1-c1-1 & e1-1-c1-1.

I also get ingress only. I understood this is an SR-SIM issue, so I hope this will be fixed in the future.
For the time being, to show both directions, either we perform 2 captures on both ends or there's a mirror service 999 that can be used for that porpose (we just need to select the ports, enable the service and then capture on the destination ingress ).

tiago-amado commented:
@hellt , thanks its working. 
Only thing is the interface syntax that has to be diferent:
const packetCaptureLinks = ['PE1-iom_e1-1-c1-1', 'PE1-iom_e1-1-c2-1'];
(...)
	function edgesharkLink(link: string) {
		let [neName, ifcName] = link.split('_');
(..)
						<div class="flex flex-col space-y-4 p-4">
							{#each packetCaptureLinks as entry}
								{@const [neName, ifcname] = entry.split('_')}
		
 So new panel version needed. I've tested built a new one and its ok

hellt commented:
I rebuilt and republished under the same v0.2.0 tag
you can repull this image and retry. If all works well I will merge it once you test it out. Thanks!

tiago-amado commented:
@hellt , Thank you! seems all good now!
